dev/tests/api-functional/testsuite/Magento/Quote/Api/CouponManagementTest.php

```diff
@@ -297,4 +297,46 @@ public function testSetMyCouponSuccess()
 
         $this->assertEquals($quoteWithCoupon->getCouponCode(), $couponCode);
     }
+
+    /**
+     * @magentoApiDataFixture Magento/Sales/_files/quote.php
+     * @magentoApiDataFixture Magento/Checkout/_files/discount_10percent.php
+     */
+    public function testSetCouponWihSpaces()
```

Comment:
Looks like this changes not related to spaces removing. Please, revert.

Reply:
Reverted.